shopinvader_search_engine_product_multi_price/models/product_product.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2024 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    shopinvader_prices = fields.Json(compute="_compute_shopinvader_prices")
+
+    @api.depends_context("index_id")
+    def _compute_shopinvader_prices(self):
+        index_id = self.env.context.get("index_id", False)
+        index = self.env["se.index"].browse(index_id)
+        for product in self:
+            prices = {"default": product._get_price()}
```

Comment:
wouldn't make more consistent to have the ID of the default pricelist here?

Reply:
I went for the simplest solution possible. Here we have no such thing as "default pricelist", the default price is the price without pricelist.

Maybe it would make even more sense to remove this "default" key (I can still add it at my project level).

For now we have no reliable solution for prices on the main branch, I think it would be great to have at lease one simple option available. Then we can build up on that or add other modules for other ways of computing and exposing prices.

I'm glad to finally see someone care about prices :grin:

Reply:
I'm testing this now and if the pricelist is already on the backend or the index we'll have a duplicated price info

Reply:
As for the default: either we add a specific field to configure which pricelist is the default one or we drop this "default" key and we assume that the client must know which pricelist to use.

Reply:
Another option: add a key `_default` pointing to the default id

Reply:
So you are testing with both `shopinvader_search_engine_product_multi_price` and `shopinvader_search_engine_product_price` (https://github.com/shopinvader/odoo-shopinvader/pull/1441) installed ?

I actually think we don't need a "default" key in `prices` of `shopinvader_search_engine_product_multi_price`, if people want a default price they can install `shopinvader_search_engine_product_price`, choose the (default) pricelist on the backend, and consider the key `price` as the default price.

We can also rename the key `prices` into `prices_by_pricelist` or `pricelist_prices` to clarify things.

Reply:
Sorry, I've been busy for the past two weeks.
I have removed the key "default" and I have renamed the key `prices` to `price_by_pricelist`.

Reply:
@simahawk